mattermost/umbrel-app.yml

```diff
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: mattermost
+name: Mattermost
+tagline: Team Chat, Open Source, Self-Hosted
+category: social
+version: "10.1.2"
+port: 8765
+description: >-
+  🚉 Mattermost is an open-source platform designed for secure collaboration throughout the software development lifecycle.
+
+
+  💬 It serves as a team messaging application that emphasizes developer productivity and the benefits of open-source software.
+
+
+  💾 Mattermost offers a self-hosted alternative to proprietary SaaS messaging services such as Slack and Microsoft Teams.
```

Comment:
In order for Mattermost to be useful it needs to be exposed to the internet, is that correct? e.g., in order for other's to use the server.

If so, then you could add a note in the app description that mentions this so that the average user isn't confused when they install the app and can't get it to work out of the box.

For example, you could add something along the lines of:

```yaml
⚠️ Mattermost needs to be exposed to the public internet in order to function as intended. The easiest way to do this is with the Cloudflare Tunnel app or Nginx Proxy Manager app from the Umbrel App Store.
```

Reply:
It for sure is of benefit if you expose it to the internet, but if you just want to use it as a local home messaging service, it would still work.

Reply:
Good point about local home messaging. I have added a note at the bottom of the description that mentions this https://github.com/getumbrel/umbrel-apps/pull/1714/commits/6054479fc1da135f55d7a23ee99ce56902623237

<img width="618" alt="image" src="https://github.com/user-attachments/assets/62d01038-f311-40a0-a4fb-4fe2eddbc96c">